CHANGELOG.md

```diff
@@ -21,6 +21,8 @@
       - Fixed an issue that would detect turning circles as sliproads
       - Fixed a bug where post-processing instructions (e.g. left + left -> uturn) could result in false pronunciations
       - Fixes a bug where a bearing range of zero would cause exhaustive graph traversals
+    - Infrastructure:
+      - Adds a feature to limit results in nearest service with a default of 100 in `osrm-routed`
```

Comment:
@daniel-j-h thanks for asking. `libosrm` should be compiled and installed by the user to be used within VROOM, so I guess this won't make a difference on my side?